source/language/types.rst

```diff
@@ -583,6 +636,17 @@ Casting specifics
 The classical types are divided into the 'standard' classical types (bool, int, 
 uint, and float) that exist in languages like C, and the 'special' classical 
 types (bit, angle, duration, and stretch) that do not.
+The standard types follow rules that mimick those of C99 for `promotion and
+conversion <https://en.cppreference.com/w/c/language/conversion>`_ in mixed
+expressions and assignments. Like the C99 specification, all float values have
+higher rank than all integer types, and higher-precision types have higher rank
+than the same type with a lower precision. Standard and special classical types
+may only mix in expressions with operators defined for those mixed types,
+otherwise explicit casts must be provided, unless otherwise noted (such as for
+assigning float values or expressions to angles).
```

Comment:
**suggestion**: Point folks to the portion of the spec where they can find more information.

Reply:
+1 - I think there's a few cases where we differ from C99 in this respect, so it'd definitely be good to put in a cross-reference.

Reply:
We discussed this in the WG today and realized that the main places we would point to are part of Jake's branch in his private repo specifying all of the operators and their input and output types, so if this PR gets merged before his he will do the necessary linking.